src/databricks/labs/ucx/assessment/jobs.py

```diff
@@ -34,6 +34,7 @@ class JobInfo:
     job_id: str
     success: int
     failures: str
+    has_jar_task: int
```

Comment:
don't add new fields to this table - it's a breaking change and every breaking change has to be justified.

rewrite it as addition to `failures` array, so that it appears in the generic failures.